packages/rfw/lib/src/flutter/material_widgets.dart

```diff
@@ -188,6 +192,45 @@ Map<String, LocalWidgetBuilder> get _materialWidgetsDefinitions => <String, Loca
     );
   },
 
+  'DropdownButton': (BuildContext context, DataSource source) {
+    final length = source.length(['items']);
+    final dropdownMenuItems = List<DropdownMenuItem<Object>>.generate(
+        length,
+        (i) => DropdownMenuItem<Object>(
```

Comment:
might be clearer if `i` was called `index` or some such
also, missing type (`(int index)`)

Reply:
Done!